src/Tools/Md2Html/Md2Html.csproj

```diff
@@ -30,32 +31,12 @@
     <PackageReference Include="Markdig">
       <Version>0.22.0</Version>
     </PackageReference>
-    <PackageReference Include="NDesk.Options">
-      <Version>0.2.1</Version>
-    </PackageReference>
   </ItemGroup>
   <Target Name="AfterBuildOps" AfterTargets="Build">
-    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'"
-		Command="if not exist $(SharedOutputPath) mkdir $(SharedOutputPath)"/>
+    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'" Command="if not exist $(SharedOutputPath) mkdir $(SharedOutputPath)" />
     <!-- the ILMergePath property points to the location of ILMerge.exe console application -->
-    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'"
-		Command="$(ILMergeConsolePath) $(OutputPath)Md2Html.exe ^
-		/allowDup ^
-		/lib:$(OutputPath) ^
-		/out:$(SharedOutputPath)Md2Html.exe ^
-		AngleSharp.Css.dll ^
-		AngleSharp.dll ^
-		HtmlSanitizer.dll ^
-		Markdig.dll ^
-		NDesk.Options.dll ^
-		System.Buffers.dll ^
-		System.Memory.dll ^
-		System.Numerics.Vectors.dll ^
-		System.Runtime.CompilerServices.Unsafe.dll ^
-		System.Text.Encoding.CodePages.dll " />
-    <Exec Condition=" '!$([MSBuild]::IsOSPlatform(`Windows`))' == 'True' "
-		Command="mkdir -p $(SharedOutputPath)"/>
-    <Exec Condition=" '!$([MSBuild]::IsOSPlatform(`Windows`))' == 'True' "
-		Command="cp $(OutputPath)*.* $(SharedOutputPath)"/>
+    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'" Command="$(ILMergeConsolePath) $(OutputPath)Md2Html.exe ^&#xD;&#xA;		/allowDup ^&#xD;&#xA;		/lib:$(OutputPath) ^&#xD;&#xA;		/out:$(SharedOutputPath)Md2Html.exe ^&#xD;&#xA;		AngleSharp.Css.dll ^&#xD;&#xA;		AngleSharp.dll ^&#xD;&#xA;		HtmlSanitizer.dll ^&#xD;&#xA;		Markdig.dll ^&#xD;&#xA;		System.Buffers.dll ^&#xD;&#xA;		System.Memory.dll ^&#xD;&#xA;		System.Numerics.Vectors.dll ^&#xD;&#xA;		System.Runtime.CompilerServices.Unsafe.dll ^&#xD;&#xA;		System.Text.Encoding.CodePages.dll " />
```

Comment:
what happened to the formatting ? Looks like it is not interpreting the new line characters properly

Reply:
That's weird, I didn't edit the .csproj manually, I just removed NDesk.Options and added CommandLineParser using the Manage NuGet packages option, if you consider it I can fix the formatting manually by editing the .csproj.

Reply:
Solved.